pallet/message-gadget/src/lib.rs

```diff
@@ -27,7 +27,7 @@ use frame_support::{log, pallet_prelude::*, traits::Get};
 use frame_system::pallet_prelude::*;
 use sp_core::{H160, H256};
 use sp_io::hashing;
-use sp_std::vec;
+use sp_std::prelude::*;
```

Comment:
@AurevoirXavier I want to keep this as `use sp_std::vec` because of this warning
```
warning: unused import: `sp_std::prelude::*`
  --> pallet/message-gadget/src/lib.rs:30:5
   |
30 | use sp_std::prelude::*;
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default
```